templates/training_job/sdk_delete_pre_build_request.go.tpl

```diff
@@ -2,5 +2,5 @@
 	// resource Unmanaged
 	latestStatus := r.ko.Status.TrainingJobStatus
 	if latestStatus != nil && *latestStatus != svcsdk.TrainingJobStatusInProgress {
-		return nil
```

Comment:
(not in scope of this PR) looks like a lot of common code can be refactored into common code